src/user/models/user-data/user-data.service.ts

```diff
@@ -201,13 +207,14 @@ export class UserDataService {
 
     await this.updateBankTxTime(slave.id);
 
-    await this.kycWebhookService.kycChanged(masterId);
-
     // reassign bank datas and users
     master.bankDatas = master.bankDatas.concat(slave.bankDatas);
     master.users = master.users.concat(slave.users);
     await this.userDataRepo.save(master);
 
+    // KYC change Webhook
+    await this.kycWebhookService.kycChanged(masterId);
+
```

Comment:
I think we should trigger the webhook on the slave, the master didn't change 🤔

Reply:
Hm right but it depends, or? If slave has KYC and master didn't..? Maybe webhook for master and slave? Or is master always the one who should have kyc?

Reply:
The slave UserData will be unused afterwards. All users and bankDatas of the slave will be transferred to the master. Even if the slave had KYC and the master didn't, the user would not have KYC after the merge.